yak/rest_user/serializers.py

```diff
@@ -62,10 +64,18 @@ class Meta:
         fields = ('client_id', 'client_secret')
 
     def get_client_id(self, obj):
-        return obj.oauth2_provider_application.first().client_id
+        # If we're using version 0.8.0 or higher
+        if oauth_toolkit_version[0] >= 0 and oauth_toolkit_version[1] >= 8:
```

Comment:
could this be abstracted into a `get_application` method that does this logic then just either gets the `client_id` or `client_secret`?